lib/resources/messages.js

```diff
@@ -43,108 +44,156 @@ export class Message extends PlivoResource {
+     * @fail {Error} return Error
+     */
+    send(optionalParams) {
+        return this.create(optionalParams);
     }
 
-    if (src && powerpackUUID) {
-      let errorText = 'Either of src or powerpack uuid, both of them are present'
-      return new Promise(function (resolve, reject) {
-        reject(new Error(errorText));
-      })
+    /**
+     * Send Message
+     * @method
+     * @param {string} src - source number
+     * @param {string} dst - destination number
+     * @param {string} text - text to send
+     * @param {object} optionalParams - Optional Params to send message
+     * @param {string} [optionalParams.type] - The type of message. Should be `sms` or `mms`. Defaults to `sms`.
+     * @param {string} [optionalParams.url] The URL to which with the status of the message is sent.
+     * @param {string} [optionalParams.method] The method used to call the url. Defaults to POST.
+     * @param {boolean} [optionalParams.log] If set to false, the content of this message will not be logged on the Plivo infrastructure and the dst value will be masked (e.g., 141XXXXX528). Default is set to true.
+     * @param {Array} [optionalParams.media_urls] For sending mms, specify the media urls in list of string
+     * @promise {object} return {@link PlivoGenericMessage} object if success
+     * @fail {Error} return Error
+     */
+    create(src, dst, text, optionalParams, powerpackUUID) {
+        var isObject = arguments.length;
+        if (isObject == 1) {
+            var powerpackUUID = src.powerpackUUID;
+            var text = src.text;
+            var dst = src.dst;
+            var url = src.url;
+            var method = src.method;
+            var type = src.type;
+            var media_urls = src.media_urls;
+            var media_ids = src.media_ids;
+            var media_ids = src.media_ids;
+            var log = src.log;
+            var trackable = src.trackable;
+            var src = src.src;
+        }
+
+        let errors = validate([{
+                field: 'dst',
+                value: dst,
+                validators: ['isRequired']
+            },
+            {
+                field: 'text',
+                value: text,
+                validators: ['isRequired']
+            },
+        ]);
+
+        if (errors) {
+            return errors;
+        }
+
+        if (!src && !powerpackUUID) {
+            let errorText = 'Neither of src or powerpack uuid present, either one is required'
+            return new Promise(function(resolve, reject) {
+                reject(new Error(errorText));
+            });
+        }
+
+        if (src && powerpackUUID) {
+            let errorText = 'Either of src or powerpack uuid, both of them are present'
+            return new Promise(function(resolve, reject) {
+                reject(new Error(errorText));
+            })
+        }
+
+        let params = optionalParams || {};
+
+        if (isObject == 1) {
```

Comment:
same if condition defined on line no 85. https://github.com/plivo/plivo-node/pull/122/files#diff-040c01672e54eef40864cf5a7b45e6d3b59df521ffe45bc5bb1ba2c6857e8b76R89

Reply:
They are different and have validated